pkg/engine/jmespath/functions.go

```diff
@@ -834,6 +834,25 @@ func jpLabelMatch(arguments []interface{}) (interface{}, error) {
 	return true, nil
 }
 
+func jpToBoolean(arguments []interface{}) (interface{}, error) {
+	if len(arguments) != 1 {
+		return nil, fmt.Errorf("jpToBoolean: expected 1 argument, got %d", len(arguments))
+	}
+	str, ok := arguments[0].(string)
+	if !ok {
+		return nil, fmt.Errorf("jpToBoolean: argument must be a string")
+	}
+	if str != "true" && str != "false" {
```

Comment:
do we want to support only `"true"` and `"false"` ?
cc @chipzoller

Reply:
Why would we need to support anything else? The input should account for all possible cases of true and false (TRUE, True, TrUe, etc).

Reply:
I asked because golang supports other values see https://www.golangprograms.com/string-to-boolean-data-type-conversion-in-go.html

Reply:
I’m not saying we should follow strictly what golang does though.
We just need to take a decision.

Reply:
@chipzoller now it's working for (TRUE, True, TrUe, etc).